index.html

```diff
@@ -1,11 +1,125 @@
+                            <option value="4">Prefer not to say</option>
+                            <option value="5">Other</option>
+                        </select>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label>How likely is that you would recommend ThinkConcept to a friend?:</label>
+                    </div>
+                    <div class="right-row">
+                        <input type="radio" name="refer" value="Definitely">Definitely<br>
+                        <input type="radio" name="refer" value="Maybe">Maybe<br>
+                        <input type="radio" name="refer" value="Not sure">Not sure
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label>What do you like most in ThinkConcept:</label>
+                    </div>
+                    <div class="right-row">
+                        <select id="dropdown">
+                            <option value="1">The indoor Design</option>
+                            <option value="2">Food Quality and Presentation</option>
+                            <option value="3">Service delivery</option>
+                            <option value="4">Price</option>
+                        </select>
+                    </div>
+                </div>
+
+                <div class="rowTab">
+                    <div class="left-row">
+                        <label>Things that should be improved in the future(Check all that apply):</label>
+                    </div>
+                    <div class="right-row">
+                        <input type="checkbox" name="future-improvement" value="Option 1">Food Menu Options<br>
+                        <input type="checkbox" name="future-improvement" value="Option 2">Payment Options<br>
+                        <input type="checkbox" name="future-improvement" value="Option 3">Number of Outlets<br>
+                        <input type="checkbox" name="future-improvement" value="Option 4">Home Delivery Service<br>
+                        <input type="checkbox" name="future-improvement" value="Option 5">Order Time and Accuracy<br>
+                        <input type="checkbox" name="future-improvement" value="Option 6">Indoor layout<br>
+                        <input type="checkbox" name="future-improvement" value="Option 7">Indoor entertainment
+                        options<br>
+                        <input type="checkbox" name="future-improvement" value="Option 8">Overall Customer Service<br>
+                        <input type="checkbox" name="future-improvement" value="Option 9">Food Packing and
+                        Presentation<br>
+                        <input type="checkbox" name="future-improvement" value="Option 10">Support amenties
+                    </div>
+                </div>
```

Comment:
# Checkbox form control

- Why don't you use semantic tags for the `.rowTab` element and its children instead of the generic tag `div` you used for grouping the label and the input controls? That would improve the overall structure of your code. Kindly see [how to build a form in HTML](https://developer.mozilla.org/en-US/docs/Learn/Forms/Your_first_form) for insights